server/@types/viewModels/index.d.ts

```diff
@@ -145,9 +145,32 @@ declare module 'viewModels' {
     responsibilities: string
   }
 
-  // TODO RR-115 - define the fields
-  // eslint-disable-next-line @typescript-eslint/no-empty-interface
-  export interface WorkInterests {}
+  export interface WorkInterests {
+    hopingToWorkOnRelease: 'YES' | 'NO' | 'NOT_SURE'
+    constraintsOnAbilityToWork: Array<
+      'CARING_RESPONSIBILITIES' | 'LIMITED_BY_OFFENSE' | 'HEALTH_ISSUES' | 'NO_RIGHT_TO_WORK' | 'OTHER' | 'NONE'
+    >
```

Comment:
Yuk! I'm surprised ktlint likes this. Let me guess, it enforces it?!

In all seriousness, I used a difference syntax for `additionalTraining` below:

```
additionalTraining?: (
      | 'CSCS_CARD'
      | 'FIRST_AID_CERTIFICATE'
...
      | 'OTHER'
      | 'NONE'
    )[]
```

We should probably use the same one. I think your approach makes it clearer that it's an array, but it's a shame about the formatting!

Reply:
Yes, the linter enforced it! 😁 
Re: approach / syntax for an array .... yeah, I dont know whats best TBH. I saw your approach in your PR, and I've also seen it in the generated types (ie. the types that get codegen'ed from the swagger specs) - so perhaps your way is more accepted/standard ??